.github/workflows/main.yml

```diff
@@ -123,8 +95,9 @@ jobs:
           pip install build
 
       - name: Install GSTools
-        env:
-          GSTOOLS_BUILD_PARALLEL: 1
+        # env:
+        #   GSTOOLS_BUILD_PARALLEL: 1
+        #   pip install --no-binary gstools-cython -v --editable .[test]
```

Comment:
Not sure about this. Should we test with parallel @LSchueler?
In the GSTools-Cython repo, we already test the cython routines with parallel execution.

Reply:
Two things come to mind.
- Depending on what kind of CI/CD resources we get from Github, the tests might finish quicker.
- Some time ago, we had very rare bugs, which where probably some kind of race conditions, only happening with parallel code execution. In case this ever happens again, we might be able to catch them earlier, if we test more in parallel.

But I'm also not sure how much weight you should give those two points.

Reply:
I think it is better to check the real world scenario where a pre-built wheel of gstools-cython is uesd. When rebuilding gstools-cython local, we could miss issues that have far more impact.

And I would argue against doing both. So for now I would be fine with only testing parallel execution in the gstools-cython repo (which should be enough IMHO).